wpilibj/src/main/java/edu/wpi/first/wpilibj/IterativeRobotBase.java

```diff
@@ -252,7 +256,7 @@ protected void loopFunc() {
       HAL.observeUserProgramAutonomous();
       autonomousPeriodic();
       m_watchdog.addEpoch("autonomousPeriodic()");
-    } else if (isOperatorControl()) {
+    } else if (isTeleop(m_controlWord)) {
```

Comment:
This should just be !isTest since you already checked the other two conditions. Then you can get rid of isTeleop.

Reply:
fixed, and added docs.